packages/manager/apps/web/client/app/hosting/database/order/components/private-database/private-database.html

```diff
@@ -0,0 +1,99 @@
+<div class="text-center" data-ng-if="$ctrl.loadingDependencies">
+    <oui-spinner data-size="l"></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loadingDependencies">
+    <p data-translate="hosting_database_order_private_description"></p>
+
+    <ovh-manager-product-offers
+        pricing-type="$ctrl.productOffers.pricingType"
+        user=":: $ctrl.productOffers.user"
+        workflow-options=":: $ctrl.productOffers.workflowOptions"
+        workflow-type=":: $ctrl.productOffers.workflowType"
+        on-success=":: $ctrl.handleSuccess(checkout)"
+        on-error=":: $ctrl.handleError(error)"
+        send-current-state=":: $ctrl.getOrderState(state)"
+    >
+        <!-- DB Type -->
+        <oui-step-form
+            data-header="{{:: 'hosting_database_order_private_step_header_type' | translate }}"
+            data-editable="$ctrl.isEditable && !$ctrl.checkoutLoading"
+        >
+            <oui-field
+                data-label="{{:: 'hosting_database_order_private_type' | translate }}"
+            >
+                <oui-radio
+                    name="orderType"
+                    data-description="{{:: 'hosting_database_order_private_type_text' | translate }}"
+                    data-model="$ctrl.order.type"
+                    data-value="'privateSQL'"
+                    data-required
+                >
+                    <span
+                        data-translate="hosting_database_order_private_type_label"
+                    ></span>
+                </oui-radio>
+            </oui-field>
+        </oui-step-form>
+
+        <!-- Customization -->
+        <oui-step-form
+            data-header="{{:: 'hosting_database_order_private_step_header_customize' | translate }}"
+            data-editable="!$ctrl.checkoutLoading"
+        >
+            <!-- Product -->
+            <oui-field
+                data-label="{{:: 'hosting_database_order_private_ram' | translate }}"
+                data-size="xl"
+            >
+                <oui-select
+                    id="product"
+                    name="product"
+                    data-model="$ctrl.order.product"
+                    data-items="$ctrl.products"
+                    data-match="label"
+                    data-placeholder="{{:: 'hosting_database_order_private_select_placeholder' | translate }}"
+                    data-on-change="$ctrl.updateVersionSelector(modelValue)"
```

Comment:
I think it's a good practice to prefix events with 'on'.
It helps developers to quickly identify events functions from others functions especially when they read controllers files.

```suggestion
                    data-on-change="$ctrl.onVersionChange(modelValue)"
```